gdal/frmts/wcs/wcsutils.cpp

```diff
@@ -645,14 +659,20 @@ CPLString GetKeywords(CPLXMLNode *root,
                 CPLString word = CPLGetXMLValue(node, nullptr, "");
                 word.Trim();
 
-                // crs, replace "http://www.opengis.net/def/crs/EPSG/0/" with EPSG:
-                const char *epsg = "http://www.opengis.net/def/crs/EPSG/0/";
-                size_t pos = word.find(epsg);
-                if (pos == 0) {
-                    CPLString code = word.substr(strlen(epsg), std::string::npos);
-                    if (code.find_first_not_of(DIGITS) == std::string::npos) {
-                        epsg_codes.push_back(atoi(code));
-                        continue;
+                // crs, replace "http://www.opengis.net/def/crs/EPSG/0/"
+                // or "urn:ogc:def:crs:EPSG::" with EPSG:
+                const char *epsg[] = {
```

Comment:
you can use ```cont char* const``` so that the array is entirely in read-only section
and same for spec array below